scripts/jenkins/Jenkinsfile

```diff
@@ -517,18 +569,20 @@ def trackCoverage(do_track, cl) {
  * @param cl A closure that should be run inside the docker image
  */
 def withDockerEnv(stage_name, image, cl) {
+    def run_stage = (stage_name =~ BUILD_TARGET) as boolean
     return [(stage_name): {
-        maybeRegexStage(stage_name) {
+        maybeStage(stage_name, run_stage) {
             node(docker_node_label) {
                 docker.withRegistry('https://hub.libelektra.org',
                                     'docker-hub-elektra-jenkins') {
                     timeout(activity: true, time: 5, unit: 'MINUTES') {
                         def cpu_count = cpuCount()
                         withEnv(["MAKEFLAGS='-j${cpu_count+2} -l${cpu_count*2}'",
                                  "CTEST_PARALLEL_LEVEL='${cpu_count+2}'"]) {
-                            echo "Staring ${env.STAGE_NAME} on ${env.NODE_NAME}"
+                            echo "Starting ${env.STAGE_NAME} on ${env.NODE_NAME}"
                             checkout scm
-                            docker.image(imageFullName(image)).inside { cl() }
+                            docker.image(imageFullName(image))
+                                  .inside("-v ${env.HOME}/git_mirrors:/home/jenkins/git_mirrors") { cl() }
```

Comment:
Please document this line.

Reply:
Yes, exactly. There are so many DSLs involved that you cannot expect people to know what this does. Please translate it into English.

I am also curious what twas wrong with the previous statement.

And please link to all these documentations in the very beginning of the file.

Reply:
there was nothing wrong with the old statement, it is just that we now want the mirror repositories available inside docker containers because we manipulate them for the releases (and we do not want full checkouts).